roles/openshift_grafana/defaults/main.yaml

```diff
@@ -6,8 +6,9 @@ openshift_grafana_proxy_image: "{{ l2_os_logging_proxy_image }}"
 openshift_grafana_state: present
 openshift_grafana_namespace: openshift-grafana
 openshift_grafana_pod_timeout: 300
-openshift_grafana_prometheus_namespace: "{{ openshift_prometheus_namespace | default('openshift-metrics') }}"
-openshift_grafana_prometheus_serviceaccount: prometheus-reader
+openshift_grafana_prometheus_namespace: "openshift-monitoring"
+openshift_grafana_prometheus_serviceaccount: "prometheus-k8s"
```

Comment:
Prometheus uses `openshift_prometheus_reader_serviceaccount_name` for that, why would we want to hardcode it here?

Reply:
I'm not sure it fits to the new cluster-monitoring changes

Reply:
This is the serviceaccount that Grafana will end up using right? In that case Grafana should have it's own service account with just the correct permission to authorize with Prometheus.

Reply:
this is the service account that playbook use to get the right token for Prometheus 
https://github.com/mrsiano/openshift-ansible/blob/b8d99a7ca5851543e30bccb4cc1f723757610f89/roles/openshift_grafana/tasks/install_grafana.yaml#L142-L144

a side note, seems like we need a read only permission as we had in the previous prometheus deployment role.

Reply:
What I'm saying is that the `ServiceAccount` called `prometheus-k8s` in the `openshift-monitoring` namespace is the `ServiceAccount` used by the Prometheus server. This `ServiceAccount` is far higher privileged than Grafana's `ServiceAccount` needs to be.

Reply:
right, previously we used something called prometheus-reader which has only read privileges
https://github.com/openshift/openshift-ansible/pull/7064

Reply:
Yeah I think it's reasonable to have the cluster monitoring stack install a cluster-role for reading purposes. This doesn't exist today, but I'll make sure this exists before feature freeze. We can move forward with this for now, and then change it when we have that available.